app/controllers/rateitapp/ratings_controller.rb

```diff
@@ -47,5 +49,16 @@ def destroy
     def rating_params
       params.permit(:user_id, :ratable_type, :ratable_id)
     end
+
+    def authenticate_user
+      access_token = bearer_token
+      Rateitapp.oauth_plugin.validate_access_token(access_token) if access_token
```

Comment:
Why not just use `bearer_token` directly instead of assigning it to a temporary variable? It seems like `bearer_token` would be a fast enough operation, and could be cached there if not.